package.json

```diff
@@ -19,24 +19,35 @@
     "lavaclient": "^4.1.1",
     "lodash-es": "^4.17.21",
     "patch-package": "^6.4.7",
+    "remove": "^0.1.5",
     "socket.io": "^4.5.2",
     "winston": "^3.8.2"
   },
   "devDependencies": {
-    "eslint": "^8.24.0"
+    "@types/crypto-js": "^4.1.1",
+    "@types/lodash-es": "^4.17.6",
+    "@typescript-eslint/eslint-plugin": "^5.39.0",
+    "@typescript-eslint/parser": "^5.39.0",
+    "eslint": "^8.24.0",
+    "rimraf": "^3.0.2",
+    "tsc-alias": "^1.7.0",
+    "tsconfig-paths": "^4.1.0",
+    "typescript": "^4.8.4"
   },
-  "main": "src/main.js",
+  "main": "dist/main.js",
   "scripts": {
+    "rimraf": "./node_modules/rimraf/bin.js",
```

Comment:
Using this script by itself doesn't work, it should be
```diff
-    "rimraf": "./node_modules/rimraf/bin.js",
+    "rimraf": "node ./node_modules/rimraf/bin.js",
```
Either way this is not needed the way I see it used.

Reply:
I have gone and applied the former.